src/lib/Components/Bidding/__stories__/BidFlow.story.tsx

```diff
@@ -19,39 +20,38 @@ const testSaleArtworkID = "5b48ed80360bca000104d057"
 const testArtworkID = "5b48ed80360bca000104d050"
 const testSaleID = "erik-reserves-moction"
 
-const selectMaxBidQuery = graphql`
-  query BidFlowSelectMaxBidRendererQuery($saleArtworkID: String!) {
-    sale_artwork(id: $saleArtworkID) {
-      ...SelectMaxBid_sale_artwork
-    }
-  }
-`
-
-const BidFlowStoryRenderer: React.SFC<any> = ({ render, query, saleArtworkID }) => {
-  return <QueryRenderer environment={createEnvironment()} query={query} variables={{ saleArtworkID }} render={render} />
-}
+const BidFlowMaxBidStoryRenderer: React.SFC<any> = ({ saleArtworkID }) => (
+  <QueryRenderer<BidFlowSelectMaxBidRendererQuery>
+    environment={createEnvironment() as any}
+    query={graphql`
+      query BidFlowSelectMaxBidRendererQuery($saleArtworkID: String!) {
+        sale_artwork(id: $saleArtworkID) {
+          ...SelectMaxBid_sale_artwork
+        }
+      }
+    `}
+    variables={{ saleArtworkID }}
+    render={renderWithLoadProgress(MaxBidScreen)}
```

Comment:
Update the type of `renderWithLoadProgress` to [that of Reaction](https://github.com/artsy/reaction/blob/fc43a81b22c391593fd94605df1f08208eebd7b9/src/Artsy/Relay/renderWithLoadProgress.tsx#L14-L17), which ensures the `$refType` of the passed in component still matches the query renderer’s `$fragmentTypes`.